src/madhuraMJun20/TestStaticNonStatic.java

```diff
@@ -0,0 +1,51 @@
+package madhuraMJun20;
+
+public class TestStaticNonStatic {
+
+	double sum(double a, double b, double c) {
+		double sumAns= a+b+c;
+		System.out.println("Sum of the marks is : " +sumAns);
+		return sumAns;
+	}
+	double average(double a, double b, double c) {
+		double avgAns= (a+b+c)/3;
```

Comment:
you should call sum method here.
double avgAns = sum(a,b,c)/3

Reply:
Done.